### PR TITLE
Moves kryo config space outside of akka

### PR DESCRIPTION
**Pull request by nvollmar**
refs #153 #63

### CODE REVIEW

**Comment by nvollmar** [comment]
was also not quite sure how to split it out. But then maybe `altoo-kryo-serialization`?
otherwise it potentially might conflict with other kryo based serializer libs.

**Comment by danischroeter** [comment]
> was also not quite sure how to split it out. But then maybe `altoo-kryo-serialization`?
> otherwise it potentially conflicts with other kryo based serializer libs.
hmm i dont think people will mix multiple kryo serializers...
most examples of akka extensions did the same: 'akka-persistence-jdbc','akka-persistence-redis','dynamodb-journal'...

File: migriation-guide.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+akka-kryo-serialization - migration guide
```
Comment (danischroeter) [inline migriation-guide.md]: filename migration-guide.md

Comment (nvollmar) [reply migriation-guide.md]: done

**Pull request merged**